manifest/kubernetes/rhel/ncp-rhel.yaml

```diff
@@ -704,12 +716,24 @@ spec:
         tier: nsx-networking
         component: nsx-ncp
         version: v1
+
+      # annotations:
+      #   prometheus.io/scrape: "true"
+      #   prometheus.io/port: "8001"
+
     spec:
       # NCP shares the host management network.
       hostNetwork: true
+
+
+      # nodeSelector is configured by operator ncp-install CRD
+      nodeSelector:
+        {{.NcpNodeSelector}}
+
       # tolerations is configured by operator ncp-install CRD
       tolerations:
-        {{ .NcpTolerations }}
+        {{.NcpTolerations}}
```

Comment:
Did you test it? I'm not sure whether `{{.NcpTolerations}}` is equal to `{{ .NcpTolerations }}`

Reply:
Yes. I've tested it, The {{.NcpTolerations}} is equal to {{ .NcpTolerations }}. we have already used this format in replicas:{{.NcpReplicas}}

Reply:
Thanks for the confirmation.